test/test_images/performance/main.go

```diff
@@ -63,6 +76,7 @@ func init() {
 	// role=aggregator
 	flag.StringVar(&listenAddr, "listen-address", ":10000", "Network address the aggregator listens on.")
 	flag.UintVar(&expectRecords, "expect-records", 1, "Number of expected events records before aggregating data.")
+	flag.Var(&additionalTags, "additional-tags", "Array of environment specific additional tags.")
```

Comment:
What is it used for?
What's the expected format?

Reply:
added an example